source/column_resizer.ts

```diff
@@ -40,45 +44,52 @@ export class ColumnResizer {
   /** Constructs a ColumnResizer.
    * @param table - The interface to the table being resized.
    */
-
   constructor(table: TableInterface) {
     this.table = table;
     this.currentIndex = -1;
     this.s0();
+    this.onMouseDown = this.onMouseDown.bind(this);
+    this.onMouseUp = this.onMouseUp.bind(this);
+    this.onMouseMove = this.onMouseMove.bind(this);
```

Comment:
Do not bind these methods from within here.

Reply:
Where should I bind them?

Reply:
You can have ```TableView``` bind them.

Reply:
That doesn't work.